src/project/export_training.py

```diff
@@ -39,8 +42,11 @@ def export_training_data(project: 'Project',
     writes exported data to the project directory
     :param project: Project from which to export training data
     :param behavior: Behavior to export
+    :param pose_version: Minimum required pose version for this classifier
```

Comment:
Are all pose versions currently backwards-compatible?

Reply:
Features are backwards compatible, but projects/classifiers can only operate on the lowest common feature set. This is the current way projects calculate this number.

Currently no logic is present for decreasing the number passed here... but it could be added (rather than just relying on param1's value).